### DISCUSSION TITLE
Openobserve log ingestion rate-limiting

### DISCUSSION THREAD

**Discussion by ep4sh** [openobserve/openobserve#4838] [Q&A] [answered]
Dear community,
I faced an issue with log ingestion in Openobserve.

My log pipeline is like the following:
**Kubernetes pods (ingress controller) -> vector.dev ->proxy (ha-proxy-1) ->** **proxy (ha-proxy-2)** -> _Openobserve._

I found that some access logs were dropped (?) during the processing, I enabled debug modes (on each step) and inspected them. 

For now I can confirm that all logs are available before Openobserve ingestion: ha-proxy-2 successfully outputs tons of them, but finally in Openobserve I could see only few of them, most of them error log, no access logs.

Are there any mechanisms that prevent logs' ingestion? it might be very clever archiver or smth else, please let me know how to debug it. Thanks!

**Comment by prabhatsharma** [comment]
No archiver. However, by default if you have more than 1000 fields in the log record the particular log line will be dropped. You can increase the number of fields using the environment variable  - ZO_COLS_PER_RECORD_LIMIT - https://openobserve.ai/docs/environment-variables/#common

**Reply by ep4sh** [reply]
Hi @prabhatsharma , thank you for prompt reply.

I assume there are no more than 50 field - I use std Kubernetes format from Vector without enrichment.
Checked out my `ZO_COLS_PER_RECORD_LIMIT` it's equal to `200`. Do you have any other ideas to check?

**Reply by prabhatsharma** [reply, marked as answer]
If one set of logs are not appearing its more likely that you have something wrong in your pipeline. Do you have any VRL functions associated that might be processing the incoming records incorrectly?

**Reply by ep4sh** [reply]
Gotcha, I've restarted my proxy pods, and now able to see the logs. 

Appreciate your time and your help! Will check why proxy did it.
